### PR TITLE
[quincy][cephadm][mix client]Add support for RHCS6 client in RHEL8

### PR DESCRIPTION
**Pull request by MohitBis**
[quincy][cephadm][mix client]Add support for RHCS6 client in RHEL8

### CODE REVIEW

**Comment by MohitBis** [comment]
Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-27EBQ4/

File: tests/cephadm/test_client.py [hunk 1]
```diff
@@ -83,6 +83,10 @@ def setup(host):
                 "8": {
                     "4": ["rhceph-4-tools-for-rhel-8-x86_64-rpms"],
                     "5": ["rhceph-5-tools-for-rhel-8-x86_64-rpms"],
+                    "6": [
```
Comment (vamahaja) [inline tests/cephadm/test_client.py:86]: As this dictionary holds live repos, don't add downstream repo here.

File: tests/cephadm/test_client.py [hunk 2]
```diff
@@ -122,7 +126,12 @@ def setup(host):
                     _node.exec_command(sudo=True, cmd=f"{enable_cmd}{repos}")
 
                 for repos in cdn_ceph_repo[rhel_version][rhcs_version]:
-                    _node.exec_command(sudo=True, cmd=f"{enable_cmd}{repos}")
+                    if rhel_version == "8" and rhcs_version == "6":
```
Comment (vamahaja) [inline tests/cephadm/test_client.py:129]: For this specific condition add comment and use repo direclty here.

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: tests/cephadm/test_client.py (hunk 1 above)
Comment (MohitBis) [reply tests/cephadm/test_client.py:87]: As discussed, Added comment

File: tests/cephadm/test_client.py (hunk 2 above)
Comment (MohitBis) [reply tests/cephadm/test_client.py:131]: As discussed, Added comment

**Review by vamahaja — APPROVED** [review]

**Pull request merged**